docs/topics/ref_known-issues-7-2-0.adoc

```diff
@@ -0,0 +1,43 @@
+:_template-generated: 2024-12-04
+:_mod-docs-content-type: REFERENCE
+
+[id="known-issues-7-2-0_{context}"]
+= Known issues
+
+This section provides highlighted known issues in {ProductFullName} version 7.2.0.
+
+.{ProductShortName} [CLI] analysis using the Container CLI fails on Windows
```

Comment:
I'd prefer `analysis using the CLI in container mode periodically fails on windows`